### PR TITLE
mgr/dashboard: Cleanup Python code

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/pybind/mgr/dashboard/module.py [hunk 1]
```diff
@@ -66,7 +66,6 @@ def fixed_init(hc_self, server, sock, makefile=CP_fileobject):
                       handle_option_command
 
 from .plugins import PLUGIN_MANAGER
-from .plugins import feature_toggles  # noqa # pylint: disable=unused-import
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/module.py]: are you really sure, we don't need this import?

Comment (votdev) [inline src/pybind/mgr/dashboard/module.py]: The ``disable=unused-import`` makes me wonder if this import is necessary here.

Comment (sebastian-philipp) [reply src/pybind/mgr/dashboard/module.py]: If you want to import something, but you only need to execute the module once (e.g. you're only registering callbacks), pyling complains at this point. 

`disable=unused-import` marks the import as valid despite of not being used.

Comment (votdev) [reply src/pybind/mgr/dashboard/module.py]: That's why i was asking. Reverted the removal.

**Pull request merged**